test/api/liskApi.js

```diff
@@ -218,460 +222,177 @@ describe('Lisk.api()', () => {
+			});
 		});
-	});
-
-	describe('#getTransaction', () => {
-		const expectedResponse = {
-			body: {
-				success: true,
-				transaction: {
-					id: '7520138931049441691',
-					height: 2346486,
-					blockId: '11556561638256817055',
-					type: 0,
-					timestamp: 25654653,
-					senderPublicKey: '632763673e5b3a0b704cd723d8c5bdf0be47e08210fe56a0c530f27ced6c228e',
-					senderId: '1891806528760779417L',
-					recipientId: '1813095620424213569L',
-					recipientPublicKey: 'e01b6b8a9b808ec3f67a638a2d3fa0fe1a9439b91dbdde92e2839c3327bd4589',
-					amount: 56340416586,
-					fee: 10000000,
-					signature: 'd04dc857e718af56ae3cff738ba22dce7da0118565675527ddf61d154cfea70afd11db1e51d6d9cce87e0780685396daab6f47cae74c22fa20638c9b71883d07',
-					signatures: [],
-					confirmations: 506685,
-					asset: {},
-				},
-			},
-		};
 
-		it('should list a defined transaction', () => {
-			const callback = sinon.spy();
-			const transactionId = '7520138931049441691';
-			const options = {
-				id: transactionId,
-			};
-			sinon.stub(LSK, 'sendRequest').callsArgWith(3, expectedResponse);
+		describe('#getBlock', () => {
+			it('should get a block of certain height', () => {
+				const blockId = '2346638';
+				const options = { height: blockId };
 
-			LSK.getTransaction(transactionId, callback);
-
-			(LSK.sendRequest.calledWith(GET, 'transactions/get', options)).should.be.true();
-			(callback.called).should.be.true();
-			(callback.calledWith(expectedResponse)).should.be.true();
-			LSK.sendRequest.restore();
+				LSK.getBlock(blockId, callback);
+				(LSK.sendRequest.calledWithExactly(GET, 'blocks', options, callback)).should.be.true();
+			});
 		});
-	});
-
-	describe('#listVotes', () => {
-		const expectedResponse = {
-			body: {
-				success: true,
-				delegates: [{
-					username: 'thepool',
-					address: '10839494368003872009L',
-					publicKey: 'b002f58531c074c7190714523eec08c48db8c7cfc0c943097db1a2e82ed87f84',
-					vote: '2317408239538758',
-					producedblocks: 13357,
-					missedblocks: 373,
-					rate: 1,
-					rank: 1,
-					approval: 21.66,
-					productivity: 97.28,
-				}],
-			},
-		};
 
-		it('should list votes of an account', () => {
-			const callback = sinon.spy();
-			const address = '16010222169256538112L';
-			const options = {
-				address,
-			};
-			sinon.stub(LSK, 'sendRequest').callsArgWith(3, expectedResponse);
-
-			LSK.listVotes(address, callback);
-
-			(LSK.sendRequest.calledWith('GET', 'accounts/delegates', options)).should.be.true();
-			(callback.called).should.be.true();
-			(callback.calledWith(expectedResponse)).should.be.true();
-			LSK.sendRequest.restore();
+		describe('#listTransactions', () => {
+			it('should list transactions of a defined account', () => {
+				const recipientAddress = '12731041415715717263L';
+				const senderAddress = '15731041415715717263L';
+				const options = {
+					recipientId: recipientAddress,
```

Comment:
This should be removed from the `options` object passed to `listTransactions` (but not the object for assertion).